### ISSUE TITLE
Blank Content PDF is generated using laravel-snappy

### ISSUE THREAD

**Issue by raheelll** [barryvdh/laravel-snappy#299]
Some times ago Laravel-snappy is working on Ubuntu server. but Now it's not working .It's working fine in local window system but on ubuntu server it's generate blank pdf. wkhtmltpdf and other extension already install on server. How can i debug this laravel-snappy PDF.

![222](https://user-images.githubusercontent.com/25636002/43772720-7925b944-9a5c-11e8-8346-2d52e3ccffcf.png)

return array(
    'pdf' => array(
        'enabled' => true,
        'binary'  => '/usr/local/bin/wkhtmltopdf-amd64',
        'timeout' => false,
        'options' => array(),
        'env'     => array(),
    ),
    'image' => array(
        'enabled' => true,
        'binary'  => '"/usr/local/bin/wkhtmltopdf-amd64"',
        'timeout' => false,
        'options' => array(),
        'env'     => array(),
    ),
);

 **How to debug this?. Where error will display.? when i print object is working correct**
Please help me ,

**Comment by LaravDev** [comment]
Try disabling any css styles, pure raw html, and see what happens.

**Comment by DotiA1** [comment]
i have a same problem, but when i disabled css its work. anyone know how to include css to pdf?

**Comment by leenooks** [comment]
Yeah, I have this problem too. I confirmed that using the Usage Example on the readme - any "internally" generated PDFs are blank, but any URL fetched PDFs are working. IE: the github line results in a github.pdf, but the bill-123.pdf and the inline pdf returns an empty PDF file (1327 bytes in size).

I do note that I had to "strip" libQt5Core.so.5 as its resulted in a missing library on buster - so I'm not sure if this is related.
